app/__init__.py

```diff
@@ -385,6 +388,26 @@ def register_v2_blueprints(application):
     application.register_blueprint(get_inbound_sms)
 
 
+def register_v3_blueprints(application):
+    """
+    Only the top level v3_blueprint should be registered directly with the application.  Register all
+    child blueprints with v3_blueprint.  This helps to standardize error handling across all v3 routes.
+    See app/v3/__init__.py.
+    """
+
+    # TODO 1360 - require auth
+    # from app.authentication.auth import validate_service_api_key_auth
+    from app.v3 import v3_blueprint
+
+    # Notifications
+    from app.v3.notifications.rest import v3_notifications_blueprint
+    # TODO 1360 - require auth
+    # v3_notifications_blueprint.before_request(validate_service_api_key_auth)
```

Comment:
Uncommenting these lines causes the /v3/notification/* routes to return a 403 error.